packages/near-membrane-base/src/validation.ts

```diff
@@ -0,0 +1,14 @@
+const TypeErrorCtor = TypeError;
+
+export function validateRequiredGlobalShapeAndVirtualizationObjects(
+    globalObjectShape: object,
+    globalObjectVirtualizationTarget: typeof globalThis
+): undefined | never {
+    if (!globalObjectShape) {
+        throw new TypeErrorCtor(`Missing global object shape.`);
+    }
+    if (!globalObjectVirtualizationTarget) {
```

Comment:
Is the plan to eventually have more realistic validation here? otherwise is seems that a simple typeof === 'object' for both is sufficient, and that can happen inline.

Reply:
I had originally written it inline like that in both near-membrane-dom and near-membrane-node (and it would eventually appear in "near-membrane-embedded") and it was just the same code repeated. If you'd prefer the repeated code, I can restore that.